content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-35-0.md

```diff
@@ -0,0 +1,703 @@
+- Fixes to any known issues from previous versions that we have in release notes. These can all be grouped under a Fixed issues H3.
+-->
+
+
+
+
+###  Spinnaker community contributions
+
+There have also been numerous enhancements, fixes, and features across all of Spinnaker's other services. See the
+[Spinnaker v1.35.5](https://www.spinnaker.io/changelogs/1.35.5-changelog/) changelog for details.
+
+## Detailed updates
+
+### Bill Of Materials (BOM)
+
+<details><summary>Expand to see the BOM</summary>
+<pre class="highlight">
+<code>artifactSources:
+  dockerRegistry: docker.io/armory
+dependencies:
+  redis:
+    commit: null
+    version: 2:2.8.4-2
+services:
+  clouddriver:
+    commit: f0e7a7470b06b6a8a007f1494bc1d97ea6d4a52c
+    version: 2.35.0
+  deck:
+    commit: b04971d387e1a0664f536427976c0d64733044b5
+    version: 2.35.0
+  dinghy:
+    commit: 50041173d1a043493409059e7fa5d7a1a80fb553
+    version: 2.35.0
+  echo:
+    commit: 6117e5941dda2f2faa52639cba5e8449b0a64d0c
+    version: 2.35.0
+  fiat:
+    commit: ce4a5e689b66fce9f5aa4e3039f0fe1af5e69f74
+    version: 2.35.0
+  front50:
+    commit: 900b169d3cea95992d781e22939bb5fa1224a75d
+    version: 2.35.0
+  gate:
+    commit: af49faafbd341109028335630f25c46e9077bc1f
+    version: 2.35.0
+  igor:
+    commit: 56f0c75e06b79f2e3029b0b0da323a2d6c0dff6f
+    version: 2.35.0
+  kayenta:
+    commit: 8c63692f834f965848a6f238035d0fb01fbc2834
+    version: 2.35.0
+  monitoring-daemon:
+    commit: null
+    version: 2.26.0
+  monitoring-third-party:
+    commit: null
+    version: 2.26.0
+  orca:
+    commit: 9e94501096394c824bbfca5be5707d0d2c5f1713
+    version: 2.35.0
+  rosco:
+    commit: 9259f32ee4c6beb1bf1da32c0161590ecf3de800
+    version: 2.35.0
+  terraformer:
+    commit: 9756bee07eaabbb25b54812996314c22554ec1c0
+    version: 2.35.0
+timestamp: "2025-01-23 23:27:17"
+version: 2.35.0
+</code>
+</pre>
+</details>
+
+### Armory
+
+
+#### Armory Rosco - 2.34.0...2.35.0
+
+  - Fix permissions on container (#675)
+  - chore(build): Updating Build dependencies (#671) (#672)
+  - chore(cd): update base service version to rosco:2024.04.18.17.06.07.release-1.34.x (#666)
+  - chore(cd): update armory-commons version to 3.16.2 (#664)
+  - chore(cd): update base service version to rosco:2024.04.18.16.53.13.master (#659)
+  - chore(cd): update base service version to rosco:2024.05.10.18.26.57.master (#665)
+  - chore(cd): update base service version to rosco:2024.06.06.14.05.00.master (#667)
+  - chore(cd): update base service version to rosco:2024.06.07.06.01.10.master (#668)
+  - chore(cd): update base service version to rosco:2024.06.12.03.37.34.master (#669)
+  - chore(cd): update base service version to rosco:2024.06.12.14.46.35.master (#670)
+  - chore(build): Updating Build dependencies (#671)
+  - chore(cd): update base service version to rosco:2024.07.01.17.46.03.master (#673)
+  - chore(cd): update base service version to rosco:2024.07.01.23.43.22.master (#674)
```

Comment:
Wondering if ... we really need all of these...

Reply:
I will remove those commit messages. I could have sworn Astrolabe was excluding them, but it turns out the pattern is missing here: https://github.com/armory-io/astrolabe/blob/master/src/main/resources/application.yml#L338-L342
Maybe it’s one of those things I kept pending during Armory days.